api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -42,11 +42,81 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 ### Data Dictonary [GET]
 
 + Response 200 (application/json)
-    + Attributes 
+    + Attributes
         + document (DataDictionary)
-    
+
+
+## Download Count [/api/v2/download/count/]
```

Comment:
This endpoint should be documented in the `contracts/search/AdvancedSearch.md` file since it is used to gauge download availability on the [Advanced Search page](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/containers/search/SearchContainer.jsx#L383), not the Download Center.